### ISSUE TITLE
Error msg: Platform zaptec does not generate unique IDs

### ISSUE THREAD

**Issue by jreidel** [custom-components/zaptec#26]
Hi,

i get this error message in HA logs:

> 2022-10-02 09:07:17.273 ERROR (MainThread) [homeassistant.components.sensor] Platform zaptec does not generate unique IDs. ID zaptec_3488b441-4403-410a-b4f5-3f9db335bf71 already exists - ignoring sensor.zaptec_charger_zap019334

Anyone know how to resove?

**Comment by Bluhme1** [comment]
Did you ever find a solution?. I've got the same error. Doesn't affect the functionality - I think

**Comment by EenerNL** [comment]
Same issue here:

```
Logger: homeassistant.components.sensor
Source: helpers/entity_platform.py:554 
Integration: Sensor (documentation, issues) 
First occurred: 12:20:39 (1 occurrences) 
Last logged: 12:20:39

Platform zaptec does not generate unique IDs. ID zaptec_14262795-2873-4e8f-b4be-0a367f9c9d27 already exists - ignoring sensor.zaptec_charger_zap015019
```

I’ve checked all sensor names. Everything looks fine. Don’t know the source of this problem.

**Comment by sveinse** [comment]
Closing this as it has been fixed in #39. Please reopen if the problem persists.
